src/Components/ComUserNav.js

```diff
@@ -23,7 +22,7 @@ export const ComUserNav = ({
             <NavLink to="/profile" className="dropdown-item" >
               Profile
             </NavLink>
-            <NavLink to="/" className="dropdown-item" onClick={() => { logOut(); setLoginState(null); }} >
+            <NavLink to="/" className="dropdown-item" onClick={() => setLoginState()} >
```

Comment:
1) pass to event handler just a function, do not create new function
```
 onClick={setLoginState}
```

2) When I click logout link, browser is reloaded. It shouldn't.